src/Notification.ts

```diff
@@ -11,11 +11,11 @@ export class Notification<T> {
   observe(observer: Observer<T>): any {
```

Comment:
Not yet, https://github.com/ReactiveX/RxJS/blob/master/src/Observer.ts#L3-L5 seems we may need those changes also. (better to include in this PR as one shot changes?)